src/main/java/utils/Config.java

```diff
@@ -0,0 +1,29 @@
+package utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class Config {
+
+    public static Properties getPropertyObject() throws IOException {
```

Comment:
You may say private here, because you don't use getPropertyObject() method outside the Config class.